src/Core.UnitTests/CSharpVB/SonarLintConfigGeneratorTests.cs

```diff
@@ -165,6 +165,24 @@ public void Generate_ValidRules_OnlyRulesFromKnownRepositoryReturned(string know
             actual.Rules.Select(r => r.Key).Should().BeEquivalentTo(new string[] { "valid1", "valid2", "valid3" });
         }
 
+        [TestMethod]
+        [DataRow("cs")]
+        [DataRow("vbnet")]
+        public void Generate_SonarSecurityRules_AreNotReturned(string language)
```

Comment:
No changes are required to the SonarLint config generator since it specifically checks for the "normal" (i.e. non-security) C# and VB analyzer ids and ignores all others, including third-party plugins packaged using the SonarQube Roslyn SDK.

Reply:
I think it's worth uniting with Generate_ValidRules_OnlyRulesFromKnownRepositoryReturned

since it's exactly the same use case.

Reply:
Sorry, I didn't notice this comment before merging.

I thought about it when writing the test and was in two minds about it. I decided in the end to have a security-specific test to match the corresponding security-specific test for the ruleset generator, but I didn't have a strong preference.

Reply:
It's ok, it's not that important. It just confused me a bit to see the tests change, even though the class didn't.

Reply:
That's why I added the review comment above...